helm_deploy/laa-crime-means-assessment/templates/external-ingress.yaml

```diff
@@ -1,63 +0,0 @@
-{{- if .Values.ingress.enabled -}}
-{{- $fullName := include "laa-crime-means-assessment.fullname" . -}}
-{{- $svcPort := .Values.service.port -}}
-{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
-apiVersion: networking.k8s.io/v1
-  {{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
-apiVersion: networking.k8s.io/v1beta1
-  {{- else -}}
-apiVersion: extensions/v1beta1
-  {{- end }}
-kind: Ingress
-metadata:
-  name: {{ $fullName }}-external
-  labels:
-    {{- include "laa-crime-means-assessment.labels" . | nindent 4 }}
-  {{- with .Values.ingress.annotations }}
-  annotations:
-    {{- toYaml . | nindent 4 }}
-  {{- end }}
-  {{- if .Values.ingress.external_whitelist_source_range }}
-    nginx.ingress.kubernetes.io/whitelist-source-range: {{ $.Values.ingress.external_whitelist_source_range }}
-    external-dns.alpha.kubernetes.io/set-identifier: {{ $fullName }}-external-{{ $.Values.ingress.environmentName}}-green
-  {{- end }}
-spec:
-  ingressClassName: {{ .Values.ingress.className }}
-  {{- if .Values.ingress.tls }}
-  tls:
-    {{- range .Values.ingress.tls }}
-    - hosts:
-        {{- range .hosts }}
-        - {{ . | quote }}
-        {{- end }}
-      secretName: {{ .secretName }}
-    {{- end }}
-  {{- end }}
-  rules:
-    {{- if .Values.ingress.externalHost }}
-    - host: {{ $.Values.ingress.externalHost.host | quote }}
-      http:
-        paths:
-          - path: {{ $.Values.ingress.externalHost.paths.swaggerPath }}
```

Comment:
This needs to remain available according to the task

Reply:
WE are no longer using external ingress.